### ISSUE TITLE
Upper eyelid location to abnormality

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10004]
**Preferred term label:**
Upper eyelid location to abnormality

**Synonyms**
Includes brow

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Hemangioma, capillary infantile, somatic

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000059
- label: Localized to upper eyelid
- parents: [Localized (HP:0012838)](https://hpo.jax.org/app/browse/term/HP:0012838)
- pmid: 
- def: Applies to an abnormality that is situated in the upper eyelid.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10004
- synonyms:
